service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -149,6 +158,14 @@ private void sendNotificationForNewNoticeOfPlacementOrder(CallbackRequest callba
                 requestData.userId())));
     }
 
+    private void triggerSendDocumentEvent(CaseDetails caseDetails, Placement placement) {
+        coreCaseDataService.triggerEvent(caseDetails.getJurisdiction(),
+            caseDetails.getCaseTypeId(),
+            caseDetails.getId(),
+            "internal-change:SEND_DOCUMENT",
+            Map.of("documentToBeSent", placement.getApplication()));
```

Comment:
story is about placement order, not application

Reply:
fixed